data/en/license_faq.yaml

```
@@ -0,0 +1,20 @@
+- Question: Is Cryptomator free to use?
+  Answer: Cryptomator is free to use on Linux, macOS and Windows, but not on iOS and Android. If you purchase a license key for one platform, it is not valid for another. A bundle license does not exist.
+
+- Question: Why is Cryptomator free to use on some platforms, but not on others?
+  Answer: Our licensing strategy differs across platforms because of constraints imposed by Apple App Store policies, Google Play Store policies, and technical differences across desktop operating systems.
+
+- Question: Do I need a supporter certificate to use Cryptomator on desktop?
+  Answer: All functional and encryption features are absolutely free to use on the desktop. By obtaining a supporter certificate, you can unlock the dark mode on desktop apps and also help accelerate Cryptomator's development. Please note, a supporter certificate is not a license key that can be used to unlock mobile apps.
+
+- Question: How can I purchase Cryptomator on Android?
+  Answer: It is a one-time purchase through Google Play or as an APK download from our website or through F-Droid. If you buy the software without going through Google Play, you will receive an Android-only license key. You cannot test the app before purchasing it, but you will be eligible for a refund if desired.
+
+- Question: How can I purchase Cryptomator on Android?
+  Answer: We use a freemium model on iOS. You can download the Cryptomator app from the App Store for free and use it in read-only mode indefinitely. A one-time in-app purchase through the Apple App Store will grant you write access.
+
+- Question: Can I avoid big-tech/gatekeepers for my purchase?
+  Answer: This is possible on Android only, as you can download the Cryptomator app as an APK file or through F-Droid and buy the license using our website. The supporter certificate can only be purchased using our website.
+
+- Question: Can I purchase Cryptomator Anonymously?
+  Answer: Yes, you can buy an Android Cryptomator license key or the Supporter certificate through https://proxysto.re. Proxystore accepts Monero, Bitcoin, and cash. You can read more about our cooperation at https://cryptomator.org/coop/proxystore/.
```

```suggestion
  Answer: Yes, you can buy a Cryptomator for Android license key or a supporter certificate through https://digitalgoods.proxysto.re/en. ProxyStore accepts Monero, Bitcoin, and cash. You can read more about our cooperation at https://cryptomator.org/coop/proxystore/.
```

Btw, you don't have to check/fix if the links are clickable or not. I can review/fix this later.